projects/core/src/configurator/commons/facade/configurator-groups.service.spec.ts

```diff
@@ -266,4 +255,46 @@ describe('ConfiguratorGroupsService', () => {
     );
     expect(configGroupUtilsService.hasSubGroups).toHaveBeenCalled();
   });
+
+  it('should get first conflict group from configuration, no conflicts', () => {
+    expect(classUnderTest.getFirstConflictGroup(productConfiguration)).toBe(
+      undefined
+    );
+  });
+
+  it('should get first conflict group from configuration', () => {
+    expect(
+      classUnderTest.getFirstConflictGroup(productConfigurationWithConflicts)
+    ).toBe(productConfigurationWithConflicts.flatGroups[0]);
+  });
+
+  it('should go to conflict solver', () => {
+    spyOn(configuratorCommonsService, 'getConfiguration').and.returnValue(
+      of(productConfigurationWithConflicts)
+    );
+    classUnderTest.goToConflictSolver(productConfigurationWithConflicts.owner);
+
+    expect(store.dispatch).toHaveBeenCalledWith(
+      new ConfiguratorActions.ChangeGroup({
+        configuration: productConfigurationWithConflicts,
+        groupId: productConfigurationWithConflicts.flatGroups[0].id,
+        parentGroupId: productConfigurationWithConflicts.groups[0].id,
+      })
+    );
+  });
+
+  it('should go to first incomplete group', () => {
```

Comment:
Here you use **incompete**!!!
Please, don't mix up **incomplete** and **uncomplete**. **Incomplete** is better because it is already used in the **model**.